modules/ROOT/pages/mongodb/mongodb-connector-6-0-design-center.adoc

```diff
@@ -0,0 +1,78 @@
+= MongoDB Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+Design Center enables you to create apps visually. To use Design Center, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Design Center:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Design Center field values, see the xref:mongodb/mongodb-connector-reference.adoc[MongoDB Connector Reference].
+
+== Configure the Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click the name of the trigger card.
+. To use the MongoDB connector as an input source trigger, choose the Object Listener input source.
++
+Alternatively, you can trigger your app using HTTP Listener or Scheduler.
++
+Design Center automatically saves changes you make in a session.
+
+== Configure the Target Component
+
+. Click *+* next to the trigger card.
+. In *Select a component*, search for the connector name.
+. Select the connector name.
+. Select an operation for the connector.
+. Enter the required values in the *General* tab.
+. If needed, enter values for other tabs.
+. Specify access information to the connector resource.
+. Click *Test Connection* to ensure you have access to MongoDB.
+
+To set up a MongoDB configuration:
++
+[%header%autowidth.spread]
+|===
+|Fields |Description
+|Servers |A location that points to the default MongoDB instance
+|Username |MongoDB username
+|Password |MongoDB password
+|Database |MongoDB database
+|Enable Metadata |Enables metadata for collections, documents, and files
+|Collection metadata samples|Adds existing collections from which to take the metadata structure
+|===
+
+You can use these operations to make a simple flow:
+
+* Insert Document:
++
+[%header%autowidth.spread]
+|===
+|Fields |Description
+|*Collection Name* |A location that points a collection to use. This collection must be created in the database that you selected previously.
+|*Fields (optional)* |Alternative way of passing fields as a literal list
+*Reviewers: It is not clear if this is an alternative way of passing fields or if it is an alternative way of passing fields as a literal list. Which is it?*
```

Comment:
This operation was finally modified, now instead of receiving fields receives the document. So this row could be replaced by:
Document - The document to insert. The `_id` object can be specified or not (and mongo will auto generate one).

Reply:
Is this wording OK?

The document to insert. If you don't use the _id field to specify an object, MongoDB automatically generates an object.

Reply:
Yes it is correct, if _id is empty, MongoDB generates an ObjectId object